MarkdownAsset.js

```diff
@@ -12,27 +12,59 @@ class MarkdownAsset extends Asset {
       linkify: true,
       typographer: true
     }).use(Meta)
-    // try loading 'markdown-it-highlight' if available
+    // try loading 'markdown-it-highlightjs' if available
     try {
-      md = md.use(require('markdown-it-highlight').default)
-    } catch (e) { }
+      md = md.use(require('markdown-it-highlightjs'), {})
+    } catch (e) {
+      if (e.code !== 'MODULE_NOT_FOUND') {
```

Comment:
if the `markdown-it-highlightjs` module is not found, will this stop the execution? I think the plugin should keep working without the syntax code highlighter

Reply:
No, the idea is that it will fail, silently, if the module is not found, and complain if it is found, but the "require" causes errors.